core/src/main/kotlin/org/evomaster/core/database/DbActionUtils.kt

```diff
@@ -345,5 +348,25 @@ object DbActionUtils {
         }
     }
 
+    fun repairFK(dbAction: DbAction, previous : MutableList<DbAction>, createdDbActions : MutableList<DbAction>,sqlInsertBuilder: SqlInsertBuilder?) : MutableList<SqlPrimaryKeyGene>{
```

Comment:
add some comments explaining what it means to repair a FK, and when it could happen that a FK is "broken"

Reply:
fixed